### DISCUSSION TITLE
Q1

### DISCUSSION THREAD

**Discussion by Madura-Prasad** [Madura-Prasad/Bumble-Bee-Loan-System#2] [Q&A] [answered]
How can I resolve the "Port 8070 required by Tomcat v8.0 Server at localhost is already in use" error when running the back-end project?

**Comment by SithumDilshan28** [comment, marked as answer]
Open your terminal or command prompt.
Enter the command netstat -ano | findstr 8070 to find the process ID (PID) that is using port 8070.
Note down the PID of the process.
Open another command prompt window with administrative privileges.
Run the command taskkill /F /PID <PID> to kill the process using the specified PID. Replace <PID> with the actual process ID you found earlier.
Once the process is terminated, you can rerun your back-end project on port 8070 without the port conflict error.
